libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotMessageHandlerBase.cs

```diff
@@ -71,7 +71,7 @@ public BotMessageHandlerBase()
                     request,
                     botFrameworkAdapter,
                     bot.OnTurnAsync,
-                    default(CancellationToken));
+                    default(CancellationToken)).ConfigureAwait(false);
```

Comment:
So technically you want to come back to the ASP.NET context here. I would not suggest using `ConfigureAwait` in any ASP.NET component ever really because, like WinForms, they are purposefully working in a specific `SynchronizationContext`... even though it's not tied to a specific thread pool per se.